report.py

```diff
@@ -0,0 +1,26 @@
+from telethon.tl.types import ChannelParticipantsAdmins
+
+
+async def init(bot):
+    # utils = modules['utils']
+
+    @bot.on(events.NewMessage(pattern='\/report'))
+    async def handler(event):
+        # mentions = '<a href="tg://user?id={}">{}</a> '.format(
+        #     event.sender_id,
+        #     utils.get_display(event.sender)
+        # )
+        if not event.reply_to_msg_id:
```

Comment:
`get_reply_message` will return `None` if there is no reply. This check can be removed and instead the `None` check added below.